symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/message/model/Message.java

```diff
@@ -134,17 +134,6 @@ public MessageBuilder data(@Nonnull Object data) {
       }
     }
 
-    /**
-     * Add attachment to the message.
-     * @param content Attachment content.
-     * @param filename Filename of the attachment.
-     * @return  this builder with the data configured.
-     */
-    public MessageBuilder addAttachment(@Nonnull InputStream content, @Nonnull String filename) {
```

Comment:
why has it been removed?

Reply:
In the PR description: 
`MessageBuilder.addAttachment(@Nonnull InputStream content, @Nonnull String filename)` removed because we want to have the same number of attachments and previews (cf APP-3136, check done on agent side).